zeek/plugin/src/plugin.cc

```diff
@@ -27,9 +24,11 @@
 #include <zeek-spicy/zeek-compat.h>
 #include <zeek-spicy/zeek-reporter.h>
 
+#ifdef HAVE_TOOLCHAIN
 namespace spicy::zeek::debug {
 const hilti::logging::DebugStream ZeekPlugin("zeek");
 }
+#endif
 
 plugin::Zeek_Spicy::Plugin SpicyPlugin;
 plugin::Zeek_Spicy::Plugin* ::plugin::Zeek_Spicy::OurPlugin = &SpicyPlugin;
```

Comment:
(commenting here since I cannot leave a comment on l.42)

When building with `BUILD_TOOLCHAIN=OFF` we do not link against hilti and `hilti::util::fmt` will be unknown. Let's use `hilti::rt::fmt` instead and include `<hilti/rt/fmt.h>`.

Reply:
Good catch!